server/src/main/java/timely/store/iterators/WritableKey.java

```diff
@@ -0,0 +1,14 @@
+package timely.store.iterators;
+
```

Comment:
This initially threw me off since Keys are Writable ( they implement Hadoop's Writable interface).